src/app/routes/__tests__/index.test.tsx

```
@@ -0,0 +1,16 @@
+import { describe, it, expect } from 'vitest';
+import { render } from '@testing-library/react';
+
+import { createRoutesStub } from 'react-router';
```

ESLint

[error] 'createRoutesStub' is defined but never used.

(@typescript-eslint/no-unused-vars)